workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Injection/WorldCommandResponseHandler.cs

```diff
@@ -0,0 +1,141 @@
+                {
+                    this.logDispatcher = logDispatcher;
+                    reserveEntityIdsDelegates = new List<Action<ReserveEntityIds.ReceivedResponse>>();
+                    createEntityDelegates = new List<Action<CreateEntity.ReceivedResponse>>();
+                    deleteEntityDelegates = new List<Action<DeleteEntity.ReceivedResponse>>();
+                }
+
+                public event Action<ReserveEntityIds.ReceivedResponse> OnReserveEntityIdsResponse
+                {
+                    add
+                    {
+                        if (!VerifyNotDisposed())
+                        {
+                            return;
+                        }
+
+                        reserveEntityIdsDelegates.Add(value);
+                    }
+                    remove
+                    {
+                        if (!VerifyNotDisposed())
+                        {
+                            return;
+                        }
+
+                        reserveEntityIdsDelegates.Remove(value);
+                    }
+                }
+
+                public event Action<CreateEntity.ReceivedResponse> OnCreateEntityResponse
+                {
+                    add
+                    {
+                        if (!VerifyNotDisposed())
+                        {
+                            return;
+                        }
+
+                        createEntityDelegates.Add(value);
+                    }
+                    remove
+                    {
+                        if (!VerifyNotDisposed())
+                        {
+                            return;
+                        }
+
+                        createEntityDelegates.Remove(value);
+                    }
+                }
+
+                public event Action<DeleteEntity.ReceivedResponse> OnDeleteEntityResponse
+                {
+                    add
+                    {
+                        if (!VerifyNotDisposed())
+                        {
+                            return;
+                        }
+
+                        deleteEntityDelegates.Add(value);
+                    }
+                    remove
+                    {
+                        if (!VerifyNotDisposed())
+                        {
+                            return;
+                        }
+
+                        deleteEntityDelegates.Remove(value);
+                    }
+                }
+
+                internal void OnReserveEntityIdsResponseInternal(
+                    ReserveEntityIds.ReceivedResponse receivedResponse)
+                {
+                    GameObjectDelegates.DispatchWithErrorHandling(receivedResponse, reserveEntityIdsDelegates,
+                        logDispatcher);
+                }
+
+                internal void OnCreateEntityResponseInternal(
+                    CreateEntity.ReceivedResponse receivedResponse)
```

Comment:
should all these methods be on two lines?

Reply:
After the namespace refactor, they now can fit into their lines, so tidying that up ✔️